src/test/java/org/folio/edge/oaipmh/MainVerticleTest.java

```diff
@@ -73,13 +78,15 @@ public static void setUpOnce(TestContext context) throws Exception {
 
     vertx = Vertx.vertx();
 
-    System.setProperty(SYS_PORT, String.valueOf(serverPort));
-    System.setProperty(SYS_OKAPI_URL, "http://localhost:" + okapiPort);
-    System.setProperty(SYS_SECURE_STORE_PROP_FILE, "src/main/resources/ephemeral.properties");
-    System.setProperty(SYS_LOG_LEVEL, "DEBUG");
-    System.setProperty(SYS_REQUEST_TIMEOUT_MS, String.valueOf(requestTimeoutMs));
+    JsonObject jo = new JsonObject()
```

Comment:
can we still test that setting these via system properties works?

Reply:
Hi @craigmcnally ,

I saw in EdgeVerticle2Test.java that properties are setted throught JsonObject but not System.setProperty(...) (like in EdgeVerticleTest.java). I implemented this test the same way. Actually, I can't guarantee now that System.setProperty(...) will work as expecting because there are many differences between EdgeVerticle and EdgeVerticle2 in properties processing - at first glance, for example, EdgeVerticle2 contains logic with merging of properties. I can investigate this question in more details and replace the settings of properties from JsonObject to System.setProperty(...). Let me please know if you consider it necessary.

Thanks.

Reply:
I think we need to be able to specify these as system properties.  If I understand the code correctly it should just work, but I think we need to know if it doesn't.

take a look at https://github.com/folio-org/edge-common/blob/8ff05c3e86accb0e88f7781341ae4c546507b3fa/src/main/java/org/folio/edge/core/Constants.java#L74 the default config values are obtained from the system properties, so like I said it should just work.

Reply:
@craigmcnally ,
to specify these as system properties we need to improve EdgeVerticle2 https://github.com/folio-org/edge-common/pull/17. Namely, we need include into defaultProperties SYS_OKAPI_URL and SYS_SECURE_STORE_PROP_FILE because at the present moment they doesn't presented in config. After that I will change JsonObject to system properties here.

Reply:
OK, sounds like a plan.  Thanks for looking into it!

Reply:
Now that the edge-common PR is merged, will you be updating this test to specify the configuration via System Proprerties?

Reply:
@craigmcnally ,
it's already done. Please, review. I'll merge if you haven't objections.

Best regards,
Slava